musicbrainz-postgres/Dockerfile

```diff
@@ -1,33 +1,23 @@
-############################################
-# Dockerfile to build musicbrainz containers
-# Based on Ubuntu 14.10
-############################################
-
-# Set the base image to Ubuntu
-FROM ubuntu:14.10
-
-# File Author / Maintainer
-MAINTAINER Rick White <rickatnight11@gmail.com>
-
-# Add extra repos
-RUN echo "deb http://us.archive.ubuntu.com/ubuntu/ utopic universe multiverse" >> /etc/apt/sources.list
-
-# Configure apt-get proxy
-#RUN echo 'Acquire::http::Proxy "http://hostname:port";' >> /etc/apt/apt.conf
-
-# Install system dependencies
-RUN apt-get update && \
-apt-get install -y adduser sudo git unzip unrar \
-bash-completion wget vim build-essential \
-postgresql-9.4 postgresql-server-dev-9.4 \
-postgresql-contrib-9.4 postgresql-plperl-9.4 \
-libjson-xs-perl libicu-dev && \
-apt-get clean autoclean && \
-apt-get autoremove -y && \
-rm -rf /var/lib/{apt,dpkg,cache,log}/ && \
-rm -rf /var/lib/postgresql
-
-# Install MusicBrainz
+FROM postgres:9.4
+MAINTAINER Paulson McIntyre <paul+docker@gpmidi.net>
+
+# Set DB params
+ENV POSTGRES_USER=musicbrainz
+ENV POSTGRES_PASSWORD=musicbrainz
+ENV POSTGRES_DB=initial
+
+# Deps
+RUN echo "deb http://http.us.debian.org/debian stable main contrib non-free" > /etc/apt/sources.list.d/nonfree.list
+RUN apt-get update
+RUN apt-get install -y adduser sudo git unzip unrar \
+        bash-completion wget vim build-essential \
+        postgresql-server-dev-9.4 \
+        postgresql-plperl-9.4 libjson-xs-perl libicu-dev \
```

Comment:
I frequently need to enter an instance during troubleshooting. Having those tools in place makes that a lot easier. Plus they don't take up much space.